### PR TITLE
feat: As a user, I want to be able to connect my wallet to Explorer and see the list of my attestations

### PR DESCRIPTION
**Pull request by OliynykPro**
## What does this PR do?

- Added my attestation list
- Add `vite-plugin-svgr` library for importing SVG as JSX

### Related ticket
https://github.com/Consensys/linea-attestation-registry/issues/414

### CODE REVIEW

File: explorer/src/components/Buttons/index.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { IButtonsProps } from "./interface";
+
+export const ButtonOutlined: React.FC<IButtonsProps> = ({ name, handler }) => {
+  return (
+    <button
+      onClick={handler}
+      className="h-12 px-4 py-3 rounded-md border border-zinc-200 justify-center items-center gap-2 inline-flex text-zinc-950 text-base font-semibold hover:border-slate-500 disabled:opacity-40"
```
Comment (voenkomatiwe) [inline explorer/src/components/Buttons/index.tsx:7]: border-zinc-200? text-zinc-950? border-slate-500? use this https://github.com/Consensys/linea-attestation-registry/blob/dev/explorer/tailwind.config.js#L45

File: explorer/src/components/Footer/index.tsx [hunk 2]
```diff
@@ -8,23 +8,23 @@ export const Footer: React.FC = () => {
...
+        <VeraxLogo />
       </Link>
       <div className="flex flex-col sm:flex-row gap-6 sm:gap-8 text-text-quaternary text-sm">
         {INFO_LIST.map(({ title, logo, url }) => (
           <a
             key={title}
             href={url}
             target="_blank"
-            className="flex justify-center self-center gap-2 hover:underline hover:text-zinc-950"
+            className="flex justify-center items-center self-center gap-2 hover:underline hover:text-zinc-950"
```
Comment (voenkomatiwe) [inline explorer/src/components/Footer/index.tsx:19]: please change the color here too

File: explorer/src/components/Header/index.tsx [hunk 3]
```diff
@@ -34,14 +34,14 @@ export const Header: React.FC<HeaderProps> = ({ isOpened, setIsOpened }) => {
     <header className="px-5 md:px-14 xl:px-[60px] py-3 justify-between items-center inline-flex">
       <div className="justify-start items-center gap-6 flex self-stretch">
         <Link to={APP_ROUTES.HOME} className="shrink-0 hover:opacity-70">
-          <img src={logo} className="h-6 xl:h-9 cursor-pointer" alt="Verax logo" />
+          <VeraxLogo />
```
Comment (voenkomatiwe) [inline explorer/src/components/Header/index.tsx:37]: You need to add the styles that were. As the screen width changes, the logo increases and decreases.
Also, check all the SVGs that you changed.

**Review by orbmis — APPROVED** [review]

File: explorer/src/utils/dateUtils.ts [hunk 4]
```diff
@@ -23,3 +23,24 @@ export const parseDateTime = (inputDate: string, isSeconds = false): ParseDateTi
...
+      : `${hoursDifference} hour${hoursDifference === 1 ? EMPTY_STRING : "s"} ago`;
+  }
+
+  const dd = String(inputDateTime.getDate()).padStart(2, ZERO_STRING);
+  const mm = String(inputDateTime.getMonth() + 1).padStart(2, ZERO_STRING);
+  const yyyy = inputDateTime.getFullYear();
+  const hours = String(inputDateTime.getHours()).padStart(2, ZERO_STRING);
+  const minutes = String(inputDateTime.getMinutes()).padStart(2, ZERO_STRING);
+
+  return `${dd}.${mm}.${yyyy}, ${hours}:${minutes}`;
```
Comment (Solniechniy) [inline explorer/src/utils/dateUtils.ts:45]: moment

**Review by Solniechniy — CHANGES_REQUESTED** [review]

File: explorer/src/components/InfoBlock/index.tsx [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import { IInfoBlockProps } from "./interface";
+import { ButtonOutlined } from "../Buttons";
+
+export const InfoBlock: React.FC<IInfoBlockProps> = ({ message, button, buttonComponent, icon }) => {
+  return (
+    <div className="h-[27.625rem] md:h-[40.875rem] flex flex-col items-center justify-center gap-6">
+      {icon}
+      <div className="text-slate-500 text-base font-normal">{message}</div>
```
Comment (voenkomatiwe) [inline explorer/src/components/InfoBlock/index.tsx:8]: text-slate-500

File: explorer/src/pages/Attestations/components/ListSwitcher/index.tsx [hunk 6]
```diff
@@ -1,40 +1,39 @@
...
-      : currentSearchParams.delete(EQueryParams.ATTESTER);
+    showMy
+      ? currentSearchParams.set(EQueryParams.SHOW_MY_ATTESTATIONS, "true")
+      : currentSearchParams.delete(EQueryParams.SHOW_MY_ATTESTATIONS);
 
     setSearchParams(currentSearchParams);
   };
 
   return (
     <div className="inline-flex bg-slate-100 gap-2 mb-6 rounded">
```
Comment (voenkomatiwe) [inline explorer/src/pages/Attestations/components/ListSwitcher/index.tsx:22]: bg-slate-100

File: explorer/src/pages/Attestations/table/columns.tsx [hunk 7]
```diff
@@ -0,0 +1,69 @@
+import { ColumnDef } from "@tanstack/react-table";
```
Comment (voenkomatiwe) [inline explorer/src/pages/Attestations/table/columns.tsx:1]: move this file to `linea-attestation-registry/explorer/src/constants/columns/myAttestation.tsx`

File: explorer/src/pages/Attestations/table/components/SortByDate/index.tsx [hunk 8]
```diff
@@ -0,0 +1,34 @@
+import { ArrowDownUp } from "lucide-react";
```
Comment (voenkomatiwe) [inline explorer/src/pages/Attestations/table/components/SortByDate/index.tsx:1]: is there any point in this component if it already exists https://github.com/Consensys/linea-attestation-registry/blob/dev/explorer/src/components/SortByDate/index.tsx

File: explorer/src/pages/Attestations/table/dataTable.tsx [hunk 9]
```diff
@@ -0,0 +1,50 @@
+import { flexRender, getCoreRowModel, useReactTable } from "@tanstack/react-table";
```
Comment (voenkomatiwe) [inline explorer/src/pages/Attestations/table/dataTable.tsx:1]: is there any point in this component if it already exists https://github.com/Consensys/linea-attestation-registry/blob/dev/explorer/src/components/DataTable/index.tsx

File: explorer/src/pages/MyAttestations/index.tsx [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+  const {
+    sdk,
+    network: { chain },
+  } = useNetworkContext();
+  const { address } = useAccount();
+  const { sm } = useWindowDimensions();
+
+  const [copied, setCopied] = useState<boolean>(false);
+
+  const handleCopy = (text: string, result: boolean) => {
```
[the remark this answers is not in the document]
Comment (OliynykPro) [reply explorer/src/pages/MyAttestations/index.tsx:32]: `CopyToClipboard` it's component from `"react-copy-to-clipboard"` library

File: explorer/src/pages/Attestation/RelatedAttestations/index.tsx [hunk 11]
```diff
@@ -1,9 +1,9 @@
 import { Attestation } from "@verax-attestation-registry/verax-sdk";
 import { EyeOffIcon } from "lucide-react";
+import { Link } from "react-router-dom";
```
Comment (voenkomatiwe) [inline explorer/src/pages/Attestation/RelatedAttestations/index.tsx:3]: here we need to return everything as it was
`import { Link } from "@/components/Link";`

**Review by Solniechniy — APPROVED** [review]

**Pull request merged**